geoips/plugins/modules/readers/geoips_netcdf.py

```diff
@@ -11,6 +11,21 @@
 interface = "readers"
 family = "standard"
 name = "geoips_netcdf"
+source_names = [
```

Comment:
Hmm, this one could be problematic, since it could support ANY geoips data type.

Reply:
Might need some sort of method for collecting all the possible supported source names within the current install, and this reader supports ANY of those sources.

Reply:
How does ``source_names = ["any"]`` sound?